earth_enterprise/src/server/geresetpgdb

```diff
@@ -279,6 +279,10 @@ if [ "$type" == "upgrade" -o "$type" == "restore" ]; then
   "$BINDIR/createdb" -T template0 geendsnippet
   "$BINDIR/createdb" -T template0 gesearch
   "$BINDIR/createdb" -T template0 gepoi
+
+  echo Adding PostGIS extension
```

Comment:
Minor: We can remove this echo. I think I originally had it in there as debug info

Reply:
Ah, nevermind. It's probably good to keep this here to verify that it's run